.github/workflows/avm.ptn.data.private-analytical-workspace.yml

```diff
@@ -0,0 +1,91 @@
+name: "avm.ptn.data.private-analytical-workspace"
+
+on:
+  schedule:
+    - cron: "0 12 1/15 * *" # Bi-Weekly Test (on 1st & 15th of month)
+  workflow_dispatch:
+    inputs:
+      staticValidation:
+        type: boolean
+        description: "Execute static validation"
+        required: false
+        default: true
+      deploymentValidation:
+        type: boolean
+        description: "Execute deployment validation"
+        required: false
+        default: true
+      removeDeployment:
+        type: boolean
+        description: "Remove deployed module"
+        required: false
+        default: true
+      customLocation:
+        type: string
+        description: "Default location overwrite (e.g., eastus)"
+        required: false
+        default: 'westeurope'
```

Comment:
```suggestion
```

If you set the default here it would be the single location used for all tests instead of using the location rotation.

Reply:
Hey @jbinko,
how come this comment was resolved? The line is still in the file. 
Also (unrelated to the issue at hand), I'd recommend to only resolve comments that are auto-resolved by GitHub (e.g., suggestions) and only comment on all other comments. This makes a re-review a lot easier :)